decidim-core/lib/decidim/component_manifest.rb

```diff
@@ -49,6 +49,10 @@ class ComponentManifest
     # The cell path to use to render the card of a resource.
     attribute :card, String
 
+    # The name of the class that handles the permissions for this component. It will
+    # probably have the form of `Decidim::<MyComponent>::Permissions`.
+    attribute :permissions_class_name, String, default: nil
```

Comment:
As I said, we should probably define a sensible default with a permissions pass-through.

Reply:
Done!